openfe/protocols/openmm_rfe/equil_rfe_methods.py

```diff
@@ -823,6 +825,30 @@ def run(self, *, dry=False, verbose=True,
         else:
             return {'debug': {'sampler': sampler}}
 
+    @staticmethod
+    def analyse(where) -> dict:
```

Comment:
Probably not for now (maybe later) - should this be moved to somewhere like openmm_utils? I'm thinking you'll probably have very similar plots for standard MD and maybe ABFEs right?

Reply:
it's more likely that you'll get `openfe_analysis <where> <simulation type>` to control which analysis is attempted.  So this might be best kept as a detail of the individual protocol, rather than trying to make a one size fits all tool for all methods